src/common/Model.tsx

```diff
@@ -180,3 +189,12 @@ export async function preClose(
     const requestId = locId.toString();
     await axios.post(`/api/loc-request/${requestId}/close`);
 }
+
+export async function preVoid(
```

Comment:
As discussed, `pre` refers to the fact that we tell the backend that a close/void operation has been or will be submitted to the blockchain so that it can immediately update its state before the operation is actually detected (generally after 10 seconds). It does not refer to the fact that the method is called before the extrinsic submission.